crates/ruff_python_parser/tests/snapshots/invalid_syntax@expressions__set__missing_closing_curly_brace_0.py.snap

```diff
@@ -0,0 +1,42 @@
+---
+source: crates/ruff_python_parser/tests/fixtures.rs
+input_file: crates/ruff_python_parser/resources/invalid/expressions/set/missing_closing_curly_brace_0.py
+---
+## AST
+
+```
+Module(
+    ModModule {
+        range: 0..47,
+        body: [
+            Expr(
+                StmtExpr {
+                    range: 46..47,
+                    value: Set(
+                        ExprSet {
+                            range: 46..47,
+                            elts: [
+                                Name(
+                                    ExprName {
+                                        range: 47..47,
+                                        id: "",
+                                        ctx: Invalid,
+                                    },
+                                ),
+                            ],
```

Comment:
Here is the same as for lists. I think we should avoid parsing out an empty expression if there's no expression in the source (an empty dictionary should remain empty, even if the closing parentheses is missing)